### PR TITLE
Add file case change recognition mention to the release notes

### PR DESCRIPTION
**Pull request by paplorinc**
Related to:
* https://github.com/gradle/gradle/pull/9663
* https://github.com/gradle/gradle/pull/9689

### CODE REVIEW

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -77,6 +77,11 @@ that take such nested zips as an input, e.g. when testing applications packaged
 The ZIP analysis now also avoids unpacking entries that are irrelevant, e.g. resource files on a compile classpath. 
 This improves performance for projects with a large amount of resource files.
 
+## File case changes are handled for `copy` and `sync` tasks
+
+On case insensitive file systems (e.g. `Windows` and `Mac`), a file/folder rename (where only the case is changed) will now be detected properly in `copy` and related tasks (e.g. `sync`).
+The task's subsequent run will not be up-to-date and won't truncate the output anymore.
```
Comment (lptr) [inline subprojects/docs/src/docs/release/notes.md]: We usually do not describe bugs fixed here, as they are mentioned in the list of closed issues. Let's skip this sentence.

Comment (paplorinc) [inline subprojects/docs/src/docs/release/notes.md]:
```suggestion
```

Comment (paplorinc) [reply subprojects/docs/src/docs/release/notes.md]: Done

File: subprojects/docs/src/docs/release/notes.md [hunk 2]
```diff
@@ -77,6 +77,11 @@ that take such nested zips as an input, e.g. when testing applications packaged
 The ZIP analysis now also avoids unpacking entries that are irrelevant, e.g. resource files on a compile classpath. 
 This improves performance for projects with a large amount of resource files.
 
+## File case changes are handled for `copy` and `sync` tasks
```
[the remark this answers is not in the document]
Comment (paplorinc) [reply subprojects/docs/src/docs/release/notes.md]: Done, thanks

File: subprojects/docs/src/docs/release/notes.md [hunk 3]
```diff
@@ -77,6 +77,11 @@ that take such nested zips as an input, e.g. when testing applications packaged
...
+
+On case insensitive file systems (e.g. `Windows` and `Mac`), a file/folder rename (where only the case is changed) will now be detected properly in `copy` and related tasks (e.g. `sync`).
```
[the remark this answers is not in the document]
Comment (paplorinc) [reply subprojects/docs/src/docs/release/notes.md]: Done, thank you

**Comment by paplorinc** [comment]
Merged, thanks for the input. If other changes are needed, let me know and I'll open a new PR.

**Pull request merged**